cmd/goal/account.go

```diff
@@ -1364,12 +1368,59 @@ func strOrNA(value *uint64) string {
 	return uintToStr(*value)
 }
 
+// DEPRECATED
+type partkeyInfo struct {
```

Comment:
given that this data structure is used only within `legacyPartkeyInfoCommand`, I would place it within the function as anonymous struct. https://go.dev/play/p/rI92HaX4_SZ

Reply:
Thanks, I was thinking of trying to reduce the scope but wasn't sure if it was too clever. The legacy function body is copy/pasted, and I want to preserve that. What do you think about defining the type inside the function?

Reply:
that's perfectly fine.